app/services/platform-apps/api/modules/display.ts

```diff
@@ -0,0 +1,118 @@
+import { Module, apiMethod, EApiPermissions, IApiContext } from './module';
+import { Display } from 'services/video';
+import uuid from 'uuid/v4';
+import electron from 'electron';
+import { Subscription } from 'rxjs/Subscription';
+
+interface IDisplayCreateOptions {
+  position: IVec2;
+  size: IVec2;
+  sourceId?: string;
```

Comment:
Why `sourceId` is optional?

Reply:
If you omit `sourceId` the display will be the entire video output.